### ISSUE TITLE
Introducing Apollo 5.5!

### ISSUE THREAD

**Issue by natashadsouza** [ApolloAuto/apollo#10599]
Apollo 5.5 enhances the complex urban road autonomous driving capabilities of previous Apollo releases, by introducing curb-to-curb driving support. With this new addition, Apollo is now a leap closer to fully autonomous urban road driving!
New upgrades include:

1. Brand new Data Pipeline Service
2. Sensor Calibration service
3. Brand new module - Storytelling
4. Scenario - Based Planning with a new planning scenarios to support curb-to-curb driving
           - Park-and-go
           - Emergency
7. Prediction Model - Caution Obstacle
           - Semantic LSTM evaluator
           - Extrapolation predictor
10. Control module
           - Model Reference Adaptive Control (MRAC)
           - Control profiling service
13. Simulation scenarios

Check it out here: https://github.com/ApolloAuto/apollo/releases/tag/v5.5.0
To learn more feel free to reach out on this thread!

**Comment by BigBenSure** [comment]
:+1: keep going!

**Comment by SenRamakri** [comment]
Good news and great job :+1:

**Comment by Martin20** [comment]
Can I ask about the installation guide for Apollo-5.5? I want to uses it along with LGSV like the tutorial existing in the link below. Do you have any prepared tutorials?

https://github.com/lgsvl/apollo-5.0

**Comment by sven-hoek** [comment]
Is there any more information on
> 2. Sensor Calibration service

? I found scripts under `/apollo/scripts` and `/apollo/modules/tools/sensor_calibration` but the former seems to use files under `modules/calibration` that don't exist (and copying the [calibration tools](https://github.com/ApolloAuto/apollo/releases/download/v2.0.0/calibration.tar.gz) ended up with missing opencv libraries and other errors) and the latter only seems to do some data preparation if I saw that correctly.
What kind of service is this and how can we use it?

Best regards!
